helpers/orderBy.js

```diff
@@ -77,4 +77,29 @@ function OrderBy (orderBy, options) {
 	};
 }
 
+/**
+ * Converts orderBy queries from string format like "field:asc"
+ * to format accepted by "json-sql" library: {field: 1}.
+ * Ascending sort method number equivalent is 1.
+ * Descending sort method number equivalent is -1.
+ * @param {string} sortQuery
+ * @param {Array} sortableFields
+ * @returns {Object}[={}] returns {} if incorrect format of sortQuery given or if field
+ */
+OrderBy.sortQueryToJsonSqlFormat = function (sortQuery, sortableFields) {
```

Comment:
The query `sortQuery` is a string and `sortableFields` is an array. Meaning user can sort only on one field from possible of multiple sortable fields. Is this a special use case? since its a normal behaviour to sort on multiple fields in same query. How to achieve it?

Reply:
Yes, sorting is possible only by one field here. Also, regexp is constructed that way.